### PR TITLE
Fix `make install` when *.hh is empty.

### PR DESCRIPTION
**Pull request by FGasper**
Issue #2387

# Description

# Review

- [ ] Add a short description of the change to the CHANGELOG.md file

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -637,7 +637,7 @@ install-wasmer:
 	install -Dm755 target/release/wasmer $(DESTDIR)/bin/wasmer
 
 install-capi-headers:
-	for header in lib/c-api/*.{h,hh}; do install -Dm644 "$$header" $(DESTDIR)/include/$$(basename $$header); done
+	shopt -s nullglob; for header in lib/c-api/*.{h,hh}; do install -Dm644 "$$header" $(DESTDIR)/include/$$(basename $$header); done
```
Comment (syrusakbary) [inline Makefile:640]: This will fail in macos (as `shopt` is not shipped there). Perhaps we can remove the `.hh` option for now

Comment (FGasper) [reply Makefile:640]: It's part of the shell on Linux. Interesting.

Comment (Hywan) [inline Makefile:640]:
```suggestion
	for header in lib/c-api/*.h; do install -Dm644 "$$header" $(DESTDIR)/include/$$(basename $$header); done
```

As we no longer ship any `.h` file, we can simplify this line as suggested.

**Review by Hywan — APPROVED** [review]
Looks good to me :-)!

bors r+

**Pull request merged**